src/styles/body.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Do you really need to reset margins and padding for all the elements?
Isn't only body enough?